README.md

```diff
@@ -24,3 +24,9 @@
 - DoB: February 24, 1999
 - Email: quisquam@email.com
 - Hobbies: Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...
+
+## Yuki Takatori
+- DoB: May 30, 1997
+- Email: yuki.takatori@g.pascalia.co.jp
+- Hobbies: tennis
```

Comment:
**[There is only an example for comment on pull request :), we do it for exercise]**
Please capitalize the first character `T` of tennis (Change `tennis` to `Tennis`)

Reply:
I'll try to fix.